relay-server/src/actors/envelopes.rs

```diff
@@ -121,8 +125,14 @@ enum ProcessingError {
     #[fail(display = "could not serialize event payload")]
     SerializeFailed(#[cause] serde_json::Error),
 
+    #[fail(display = "could not build envelope for upstream")]
+    EnvelopeBuildFailed(#[cause] EnvelopeError),
+
+    #[fail(display = "could not encode request body")]
+    BodyEncodingFailed(#[cause] std::io::Error),
```

Comment:
would map all of this to None for now, I think this change makes sense but if we want to do that I would reevaluate Self::UpstreamRequestFailed too

Reply:
done in 8fca0ef